connectors/.env.example

```diff
@@ -0,0 +1,5 @@
+
+API_PORT=3002
```

Comment:
Ideally we align with the other next-based services and this lives in the package.json with a `-p` argument that you add support for in the server binary.